base/show.jl

```diff
@@ -197,8 +220,7 @@ function show_datatype(io::IO, x::DataType)
     # and `true` if we are printing type parameters outside a method signature.
```

Comment:
I believe this comment can be invalidated now (it effectively was attempting to compute free-type-vars)